lxd/main_cluster.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	lxd "github.com/lxc/lxd/client"
+	"github.com/lxc/lxd/lxd/cluster"
+	"github.com/lxc/lxd/lxd/db"
+	"github.com/lxc/lxd/lxd/sys"
+	"github.com/lxc/lxd/shared"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type cmdCluster struct {
+	global *cmdGlobal
+}
+
+func (c *cmdCluster) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = "cluster"
+	cmd.Short = "Low-level cluster administration commands"
+	cmd.Long = `Description:
+  Low level administration tools for inspecting and recovering LXD clusters.
+`
+	// List database nodes
+	listDatabaseNodes := cmdClusterListDatabaseNodes{global: c.global}
+	cmd.AddCommand(listDatabaseNodes.Command())
+
+	// Recover
+	recover := cmdClusterRecoverFromQuorumLoss{global: c.global}
+	cmd.AddCommand(recover.Command())
+
+	return cmd
+}
+
+type cmdClusterListDatabaseNodes struct {
+	global *cmdGlobal
+}
+
+func (c *cmdClusterListDatabaseNodes) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = "list-database-nodes"
+	cmd.Aliases = []string{"ls"}
+	cmd.Short = "Print the addresses of the cluster members serving as database nodes"
```

Comment:
Ok, so maybe just `list-database` and change from `cluster members serving as database nodes` to `cluster members serving the database`?